src/utils/fireQuery.ts

```diff
@@ -553,6 +553,8 @@ const getEnglishQuery = (args: FireQueryArgs) => {
                   Object.entries(output).forEach(([k, v]) => {
                     p[label + k] = v;
                   });
+                } else if (typeof output === "number") {
+                  p[label] = output.toString();
```

Comment:
I'm concerned that this may break sorting - if you have three results: 1, 11, 2, how does that sort with this change?

Reply:
Ah good thought.
Apparently it does sort.

No sort
![image](https://user-images.githubusercontent.com/3792666/215012637-ab8a17fc-15a2-4a86-9469-3d3ad83e2e31.png)

Descending
![image](https://user-images.githubusercontent.com/3792666/215012745-cc36ec29-90d2-428f-a20b-2e3cacd5e677.png)

Ascending
![image](https://user-images.githubusercontent.com/3792666/215012672-bd5807ac-eea1-45ce-b065-3b21e7a84c27.png)

Reply:
Sounds good!